BUILD

```diff
@@ -29,3 +29,35 @@ Maven repository for it. To build the binding:
         -Dpackaging=jar
 
 3. Uncomment `<module>nosqldb</module>` and run `mvn clean package`.
+
+# Mapkeeper 
+
+Mapkeeper binding doesn't get built by default because there is no
+Maven repository for it. To build the binding:
+
+1. follow "https://github.com/m1ch1/mapkeeper/wiki/Getting-Started" to Generate Thrift MapKeeper Binding
+   a.  install libevent
+   b.  install boost
+   c.  install thrift
+   d.  generate mapkeeper binding
+
+2. After generating mapkeeper binding, go the "mapkeeper/thrift/gen-java/target", and 
```

Comment:
yeah, I know that, just mvn install for mapkeeper

在 24/5/15 10:28 pm, Sean Busbey 写道:
>
> In BUILD 
> <https://github.com/brianfrankcooper/YCSB/pull/262#discussion_r30952819>:
>
> > @@ -29,3 +29,35 @@ Maven repository for it. To build the binding:
> >          -Dpackaging=jar
> >
> >  3. Uncomment `<module>nosqldb</module>` and run `mvn clean package`.
> > +
> > +# Mapkeeper
> > +
> > +Mapkeeper binding doesn't get built by default because there is no
> > +Maven repository for it. To build the binding:
> > +
> > +1. follow "https://github.com/m1ch1/mapkeeper/wiki/Getting-Started" to Generate Thrift MapKeeper Binding
> > +   a.  install libevent
> > +   b.  install boost
> > +   c.  install thrift
> > +   d.  generate mapkeeper binding
> > +
> > +2. After generating mapkeeper binding, go the "mapkeeper/thrift/gen-java/target", and
>
> There doesn't need to be a remote repo to install locally. Just use 
> "mvn install" instead of "mvn package". Then other builds on the same 
> machine will see the artifacts.
>
> If you do set up a repo, please track your activity on #181 
> <https://github.com/brianfrankcooper/YCSB/issues/181>.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/brianfrankcooper/YCSB/pull/262/files#r30952819>.
>

-- 

Best Regards !

*PanFengfeng

*

State Key Laboratory of Computer Architecture,

Institute of Computing Technology,

Chinese Academy of Sciences

Address:

1005D,No 6. Kexueyuan South Road

Haidian District, Beijing, China, 100190

Email:

panfengfeng@ict.ac.cn <mailto:panfengfeng@ict.ac.cn> or 
panfengfeng@ncic.ac.cn <mailto:panfengfeng@ncic.ac.cn>